tsconfig.test.json

```diff
@@ -1,13 +1,14 @@
 {
 	"extends": "./tsconfig.base.json",
 	"compilerOptions": {
-		"module": "commonjs",
+		"module": "CommonJS",
+		"moduleResolution": "Node",
```

Comment:
Happened to see this PR in the release notes, and skimmed through it as I was curious 🙃.

Is there a reason to use the "Node" moduleResolution? https://www.typescriptlang.org/tsconfig/#moduleResolution (It's deprecated in favor of the equivalent Node10, which is itself discouraged)

Reply:
Actually, there was an issue with other moduleResolution values, there is a bulk of issues on mocha side. As a rude example - [this](https://github.com/mochajs/mocha/issues/5171). But now tests building is moved to esbuild and now moduleResolution is not considered when building. It can be safely removed. /cc @eamodio 

https://github.com/gitkraken/vscode-gitlens/pull/3648